privaterelay/middleware.py

```diff
@@ -15,3 +23,30 @@ def __call__(self, request):
 
         request.fxa_account = fxa_account
         return self.get_response(request)
+
+
+def _get_metric_view_name(request):
+    if request.resolver_match:
+        view = request.resolver_match.func
+        return f'{view.__module__}.{view.__name__}'
+    return '<unknown_view>'
```

Comment:
Metrics keys should be alphanumeric and period and underscore:

https://markus.readthedocs.io/en/latest/metricsoverview.html#stats

Reply:
Oops--my bad. This is being used as a tag value. I think I'd do the same with tag values.

https://markus.readthedocs.io/en/latest/metricsoverview.html#using-tags